data-service/src/test/java/com/amazonaws/blox/dataservice/repository/ListClustersTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataservice.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.groups.Tuple.tuple;
+
+import com.amazonaws.blox.dataservice.model.EnvironmentId;
+import com.amazonaws.blox.dataservice.test.data.ModelBuilders;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+public class ListClustersTest extends EnvironmentRepositoryTestCase {
+  private static final String CLUSTER = "ClusterOne";
+  private static final String OTHER_CLUSTER = "ClusterTwo";
+  private static final String ENVIRONMENT_NAME = "EnvironmentOne";
+  private static final String OTHER_ENVIRONMENT_NAME = "EnvironmentTwo";
+  ModelBuilders models = ModelBuilders.builder().build();
+
+  List<EnvironmentId> ids =
+      Arrays.asList(
+          models.environmentId("1", "ClusterOne", "EnvironmentOne"),
+          models.environmentId("2", "ClusterTwo", "EnvironmentTwo"),
+          models.environmentId("2", "ClusterThree", "EnvironmentThree"));
+
+  public void givenEnvironments(List<EnvironmentId> ids) throws Exception {
```

Comment:
should be private? Would be better to move at the end of the class, as it's helper methods.